internal/pkg/workflows/template/pull.yml.tmpl

```diff
@@ -16,6 +16,8 @@ jobs:
         uses: actions/checkout@v2
       - name: Install Keboola as Code CLI
         uses: ./.github/actions/install
+        with:
+          githubToken: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Ten token tam sice není nutně potřeba, ale vyřeší to případný problémy s GH API rate limitama.